docs/source/apps/proj.rst

```diff
@@ -172,6 +172,13 @@ also used for supporting files like datum shift files.
     Usage of *+opt* varies with projection and for a complete description
     consult the :ref:`projection pages <projections>`.
 
+.. versionadded:: 9.3.0
+
+    *{crs}* is one of the possibilities accepted by proj_create(), provided it
```

Comment:
```suggestion
    *{crs}* is one of the possibilities accepted by :c::func:`proj_create()`, provided it
```

Reply:
Further down the page an example using proj with a CRS would be nice as well

Reply:
Done